cargo-crev/src/main.rs

```diff
@@ -259,11 +259,24 @@ fn run_command(command: opts::Command) -> Result<CommandExitStatus> {
                 }
             }
             opts::Id::Trust(args) => {
-                create_trust_proof(
```

Comment:
I don't understand why is this the only deleted code. :D . Seems like there should be more red color?

Reply:
Yes! :) I wanted to do the minimal amount of work to get the general direction/idea across first. Will do more work on this PR.